src/components/speiling/sykmeldinger/sykmeldingstatuspanel/SykmeldingStatuspanelOpplysning.tsx

```diff
@@ -134,7 +136,7 @@ interface SykmeldingopplysningFravaersperioderProps {
 
 export const SykmeldingopplysningFravaersperioder = (
   sykmeldingopplysningFravaersperioderProps: SykmeldingopplysningFravaersperioderProps
-) => {
+): ReactElement | null => {
```

Comment:
Kan kanskje returnere <> i stedet for null, så vil returtypen alltid være ReactElement?

Reply:
Ja, fikser